### DISCUSSION TITLE
Fontbakery 0.11.0: jinja2 error (fontbakery.reporters not understandable)

### DISCUSSION THREAD

**Discussion by eweracs** [fonttools/fontbakery#4476] [Q&A] [answered]
Hello, this morning, I updated to Fontbakery 0.11.0. Unfortunately, Fontbakery is unable to generate the HTML reports as it runs into an issue with Jinja2. I updated Jinja2 to the latest version.

`ValueError: The 'fontbakery.reporters' package was not installed in a way that PackageLoader understands.`

Is there anything I can do on my end? I have currently downgraded to Fontbakery 0.10.9, which works fine.
OS: macOS 14.2.1
Python: 3.10.2
Jinja2: 3.1.3

Thanks for any pointers!

**Comment by m4rc1e** [comment]
Have you tried creating a fresh virtual environment and then reinstalling fontbakery?

Steps are:

```BASH
python3 -m venv venv
source venv/bin/activate
pip install fontbakery[googlefonts]
```

Apologies if this sounds obvious. @simoncozens is doing a lot of refactoring work so you may have a lot of conflicts if you're just upgrading small parts.

**Reply by eweracs** [reply]
Thanks for the instructions, but I'm afraid that didn't work. I get the same error.

**Comment by simoncozens** [comment]
I wonder if it's something about fontbakery being packaged as a wheel or something like that.

**Reply by simoncozens** [reply]
Yep, I'm a numpty, we're not including the template directory in the wheel.

**Comment by simoncozens** [comment, marked as answer]
0.11.1 should now fix this.
